### PR TITLE
OSDOCS-12080: Documented the 4.17 RN for the nmstate Operator metrics

### PR DESCRIPTION
**Pull request by dfitzmau**
Engineer approvals on https://github.com/openshift/openshift-docs/pull/75781

Version(s):
4.17

Issue:
[OSDOCS-12080](https://issues.redhat.com/browse/OSDOCS-12080)

Link to docs preview:
[View metrics collected by the Kubernetes NMState Operator](https://82212--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-networking-nmstate-operator-metrics_release-notes)

### CODE REVIEW

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -470,6 +470,13 @@ Red{nbsp}Hat support exists for using the Kubernetes NMState Operator on {azure-
 
 For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator_k8s-nmstate-operator[About the Kubernetes NMState Operator].
 
+[id="ocp-4-17-networking-nmstate-operator-metrics_{context}"]
+==== View metrics collected by the Kubernetes NMState Operator
+
+The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect metrics from the `kubernetes_nmstate_features_applied` component and expose them as ready-to-use metrics. Both *Administrator* and *Developer* perspectives support viewing these metrics.
```
Comment (xenolinux) [inline release_notes/ocp-4-17-release-notes.adoc:476]:
```suggestion
The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect metrics from the `kubernetes_nmstate_features_applied` component and expose them as ready-to-use metrics. You can view these metrics by using  the *Administrator* and *Developer* perspectives.
```

**Review by xenolinux — COMMENTED** [review]
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

One suggestion; otherwise LGTM

File: release_notes/ocp-4-17-release-notes.adoc [hunk 2]
```diff
@@ -470,6 +470,13 @@ Red{nbsp}Hat support exists for using the Kubernetes NMState Operator on {azure-
 
 For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator_k8s-nmstate-operator[About the Kubernetes NMState Operator].
 
+[id="ocp-4-17-networking-nmstate-operator-metrics_{context}"]
+==== View metrics collected by the Kubernetes NMState Operator
+
+The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect metrics from the `kubernetes_nmstate_features_applied` component and expose them as ready-to-use metrics. You can view these metrics by using  the *Administrator* and *Developer* perspectives.
```
Comment (xenolinux) [inline release_notes/ocp-4-17-release-notes.adoc:476]:
```suggestion
The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect metrics from the `kubernetes_nmstate_features_applied` component and expose them as ready-to-use metrics. You can view these metrics by using the *Administrator* and *Developer* perspectives.
```
Sorry, my suggested change too had an extra space, which needs to be removed.

Comment (dfitzmau) [reply release_notes/ocp-4-17-release-notes.adoc]: Thanks. Updated!

File: release_notes/ocp-4-17-release-notes.adoc (hunk 1 above)
Comment (dfitzmau) [reply release_notes/ocp-4-17-release-notes.adoc]: :heavy_check_mark:

**Pull request merged**